### PR TITLE
Add setting dialog for user id on macOS test app

### PR DESCRIPTION
**Pull request by charmian01**
Things to consider before you submit the PR:

* [ ] Has `CHANGELOG.md` been updated?
* [X] Are tests passing locally?
* [X] Are the files formatted correctly?
* [ ] Did you add unit tests?
* [ ] Did you check UI tests on the sample app? They are not executed on CI.
* [ ] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description
Add the setting dialog to set user id on macOS test app

Screen Shot 2019-07-01 at 19 34 39

## Related PRs or issues
Bug [63297](https://msmobilecenter.visualstudio.com/Mobile-Center/_queries/edit/63297/?triage=true)

### CODE REVIEW

**Comment by jaelim-ms** [comment]
Looks good to me. Need to test the test app.

File: SasquatchMac/SasquatchMac/ViewControllers/AppCenterViewController.swift [hunk 1]
```diff
@@ -223,11 +217,51 @@ class AppCenterViewController : NSViewController, NSTextFieldDelegate, NSTextVie
...
+      UserDefaults.standard.removeObject(forKey: kMSUserIdKey)
+      appCenter.setUserId(nil)
+      break
+    case NSAlertSecondButtonReturn:
+      let text = textView.string ?? ""
+      UserDefaults.standard.set(text, forKey: kMSUserIdKey)
+      appCenter.setUserId(text)
+      break
+    case NSAlertThirdButtonReturn:
+      break
```
Comment (MatkovIvan) [inline SasquatchMac/SasquatchMac/ViewControllers/AppCenterViewController.swift]: Extra case

File: SasquatchMac/SasquatchMac/ViewControllers/AppCenterViewController.swift [hunk 2]
```diff
@@ -223,11 +217,51 @@ class AppCenterViewController : NSViewController, NSTextFieldDelegate, NSTextVie
...
+
   private func prodLogUrl() -> String {
     return startUpModeForCurrentSession == StartupMode.OneCollector.rawValue ? ocProdLogUrl : acProdLogUrl
   }
   
   private func prodAppSecret() -> String {
     return startUpModeForCurrentSession == StartupMode.OneCollector.rawValue ? "" : (UserDefaults.standard.object(forKey: kMSAppSecret) ?? appCenter.appSecret()) as! String
   }
+
+  func showUserId () -> String {
```
Comment (MatkovIvan) [inline SasquatchMac/SasquatchMac/ViewControllers/AppCenterViewController.swift]:
```suggestion
  func showUserId() -> String {
```

File: SasquatchMac/SasquatchMac/ViewControllers/AppCenterViewController.swift (hunk 1 above)
Comment (charmian01) [reply SasquatchMac/SasquatchMac/ViewControllers/AppCenterViewController.swift]: Removed it.

**Comment by jaelim-ms** [comment]
Why is the reset button on the right as a default?

**Comment by jaelim-ms** [comment]
Tested the functionality that works as expected.

**Comment by jaelim-ms** [comment]
Ah.. All dialogs have reset buttons as default.

**Pull request merged**